src/runtime/4.8/windowsservercore-2004/Dockerfile

```diff
@@ -8,14 +8,6 @@ ENV `
     COMPLUS_NGenProtectedProcess_FeatureEnabled=0
 
 RUN `
-    # Apply latest patch
-    curl -fSLo patch.msu http://download.windowsupdate.com/d/msdownload/update/software/secu/2021/01/windows10.0-kb4601050-x64-ndp48_04c8736f250c06b3787d7d9ec366f011c6f8ca91.msu `
-    && mkdir patch `
-    && expand patch.msu patch -F:* `
-    && del /F /Q patch.msu `
-    && dism /Online /Quiet /Add-Package /PackagePath:C:\patch\windows10.0-kb4601050-x64-ndp48.cab `
-    && rmdir /S /Q patch `
-    `
```

Comment:
Is the `&& %windir%\Microsoft.NET\Framework\v4.0.30319\ngen update` call necessary anymore?  I am wondering if this was only necessary because of the patch application?

Reply:
Fixed, PTAL